pilot/pkg/config/kube/gateway/testdata/route-precedence.yaml.golden

```diff
@@ -138,7 +138,7 @@ spec:
   gateways:
   - mesh
   hosts:
-  - a.domain.example
+  - a-example.allowed-1.svc.domain.suffix
```

Comment:
I don't think we are following https://gateway-api.sigs.k8s.io/geps/gep-1426/#hostnames-field

Reply:
For mesh routes, I'm completely ignoring the hostnames field and generating the virtual service host based on the referenced service only. That's how I interpret https://gateway-api.sigs.k8s.io/geps/gep-1426/#hostnames-field. Am I missing something?

Reply:
Github won't let me commento n the exact line but it looked like a.domain.example was showing up earlier?

Reply:
Oh that was a Gateway one, nvm. :+1: